canonical/snapcraft.io
# Update dependencies

## Done
Updated major dependencies based on [this renovate PR](https://github.com/canonical/snapcraft.io/pull/4183)

## How to QA
- Go to https://snapcraft-io-4217.demos.haus/
- Click around and check if anything seems broken
  - Particularly check https://snapcraft-io-4217.demos.haus/admin and https://snapcraft-io-4217.demos.haus/<snap_name>/listing

## Issue
This will resolve the PR linked to above
